datasets/norec/norec.py

```diff
@@ -0,0 +1,117 @@
+# coding=utf-8
+# Copyright 2020 HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+import glob
+import os
+
+import conllu
+
+import datasets
+
+
+_CITATION = """\
+NoReC: The Norwegian Review Corpus
+Erik Velldal, Lilja Øvrelid, Eivind Alexander Bergem, Cathrine Stadsnes, Samia Touileb, Fredrik Jørgensen
+2018
+http://www.lrec-conf.org/proceedings/lrec2018/pdf/851.pdf
+"""
+
+_DESCRIPTION = """\
+NoReC was created as part of the SANT project (Sentiment Analysis for Norwegian Text), a collaboration between the Language Technology Group (LTG) at the Department of Informatics at the University of Oslo, the Norwegian Broadcasting Corporation (NRK), Schibsted Media Group and Aller Media. This first release of the corpus comprises 35,194 reviews extracted from eight different news sources: Dagbladet, VG, Aftenposten, Bergens Tidende, Fædrelandsvennen, Stavanger Aftenblad, DinSide.no and P3.no. In terms of publishing date the reviews mainly cover the time span 2003–2017, although it also includes a handful of reviews dating back as far as 1998.
+"""
+
+_URL = "https://www.mn.uio.no/ifi/english/research/projects/sant/data/norec/norec-1.0.1.tar.gz"
+_TRAIN = "conllu/train"
+_DEV = "conllu/dev"
+_TEST = "conllu/test"
+
+
+class NoReC(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Norec(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `norec`